diff --git a/src/runtime/moduleobject.cs b/src/runtime/moduleobject.cs
index 7389855..c2ddf56 100644
--- a/src/runtime/moduleobject.cs
+++ b/src/runtime/moduleobject.cs
@@ -35,7 +35,10 @@ namespace Python.Runtime
             string docstring = "Namespace containing types from the following assemblies:\n\n";
             foreach (Assembly a in AssemblyManager.GetAssemblies(name))
             {
-                filename = a.Location;
+                if (!a.IsDynamic && a.Location != null)
+                {
+                    filename = a.Location;
+                }
                 docstring += "- " + a.FullName + "\n";
             }
 
